lnst/Devices/Device.py

```diff
@@ -855,7 +855,8 @@ def tx_pause_frames(self, value):
 
     def _read_pause_frames(self):
         try:
-            res, _ = exec_cmd("ethtool -a %s" % self.name)
+            res, _ = exec_cmd("ethtool -a %s" % self.name,
+                              log_outputs = False)
```

Comment:
with my previous comment i think this doesn't have to change, or is it a problem that the failed messages are also part of the `debug` logs?

on the other hand we may want to consider adding a debug log to the exception handler... if this REALLY interests us.